test/qptest.jl

```diff
@@ -31,7 +31,7 @@ end
 
 @testset "PosDef QP" begin
     n = 1000
-    obj♯ = -193.05853878066543
+    obj♯ = -175.028369041816
```

Comment:
Is it OK to simply change the value? And, it is not a good idea to randomly generate instance. See below:
```
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.7.2 (2022-02-06)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

julia> using Random

julia> Random.seed!(1)
TaskLocalRNG()

julia> randn(3)
3-element Vector{Float64}:
 -0.07058313895389791
  0.5314767537831963
 -0.806852326006714
```

```
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.6.0 (2021-03-24)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

julia> using Random

julia> Random.seed!(1)
rMersenneTwister(1)

julia> randn(3)
3-element Vector{Float64}:
  0.2972879845354616
  0.3823959677906078
 -0.5976344767282311
```

Instead, we can pre-generate the parameters `P`, `q`, `u`, `l`.

Reply:
good catch. I think pre-generating the parameters would solve this issue indeed. The other solution is to compare with the solution given returned by a different solver (JuliaSmoothOptimizers has an implementation of tron too)

Reply:
I don't really understand. This test is to compare against the TRON implementation here, which I did for these particular values and updated the objective value with what TRON printed out (-175...).

Actually, it was not so hard to build a shared library of TRON and then call it here through ExaTron. Maybe we can do that as part of the CI. But let's move forward for now.